addons/sale/models/sale_order_line.py

```diff
@@ -110,7 +110,7 @@ class SaleOrderLine(models.Model):
         compute='_compute_product_uom_qty',
         digits='Product Unit of Measure', default=1.0,
         store=True, readonly=False, required=True, precompute=True)
-    product_uom = fields.Many2one(
+    product_uom_id = fields.Many2one(
```

Comment:
Why `product_uom_id` and not `uom_id` ?
Honestly seems way more consistent to use `uom_id` everywhere, and to only use a custom name when there are multiple uoms fields on the same record.

Reply:
Mainly because `product_uom_id` is used more often as a field than `uom_id` (27 `product_uom_id = fields` vs 8 `uom_id = fields` in master).
Now seeing as this PR impacts a lot of stuff, it will probably be resized to only impact stock/mrp modules

Reply:
Since it's a cleanup task, either do it the best global way (guidelines > existing code) or don't do it in other scopes yes :+1:.